cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResources.java

```diff
@@ -150,18 +163,33 @@ public void deleteDiskEncryptionSet(DiskEncryptionSetDeletionRequest diskEncrypt
         }
     }
 
+    private void checkAndDeleteDesResourceGroupName(AzureClient azureClient, String desResourceGroupName) {
+        Matcher matcher = Pattern.compile(".*-CDP_DES-.*").matcher(desResourceGroupName);
+        if (matcher.matches()) {
+            LOGGER.info("Deleting resource group \"{}\".", desResourceGroupName);
+            azureClient.deleteResourceGroup(desResourceGroupName);
```

Comment:
Irrespective of the existence of the RG on Azure side, `persistenceNotifier.notifyDeletion()` should be invoked for the `CloudResource` representing the DES RG.